src/assets/code_of_conduct.md

```diff
@@ -0,0 +1,93 @@
+<p align="center">
+  <img width="300" height="300" src="./event-logo.svg">
+</p>
+
+<center>
+
+  # Code of Conduct
+</center>
```

Comment:
These `<center>` tags still don't do anything, update to use the same as the image.

Reply:
Let's see, it's not making it easy to play with headers unfortunately :(